From: Christopher Huhn <C.Huhn@gsi.de>
Date: Fri, 6 Nov 2015 17:18:51 +0100
Subject: Tweak default paths for Debian compliance

---
 core/lib/Foswiki.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/lib/Foswiki.spec b/core/lib/Foswiki.spec
index 7a3688e..b4c56f0 100644
--- a/core/lib/Foswiki.spec
+++ b/core/lib/Foswiki.spec
@@ -185,7 +185,7 @@ $Foswiki::cfg{TemplateDir} = '/usr/share/foswiki/templates';
 #    * ={WorkingDir}/registration_approvals= - this is used by the
 #      default Foswiki registration process to store registrations that
 #      are pending verification.
-$Foswiki::cfg{WorkingDir} = '/var/run/foswiki';
+$Foswiki::cfg{WorkingDir} = '/var/spool/foswiki';
 
 # **PATH LABEL="Safe PATH" CHECK='undefok'**
 # You can override the default PATH setting to control
